lib/generators/curation_concerns/install_generator.rb

```diff
@@ -104,5 +105,12 @@ def resource_types_config
     def simple_form_bootstrap
       generate 'simple_form:install --bootstrap'
```

Comment:
Things appear to work fine without those files so I removed that line and changed the `desc`.